include/nbl/builtin/hlsl/spirv_intrinsics/core.hlsl

```diff
@@ -119,6 +180,44 @@ template<typename Signed>
 [[vk::ext_instruction( spv::OpBitFieldSExtract )]]
 Signed bitFieldSExtract( Signed val, uint32_t offsetBits, uint32_t numBits );
 
+}
+
+// should be in `nbl::hlsl::spirv` but would collide right now
```

Comment:
use `enable_if` with checks on `U` and `T` being `pointer_t<PhysicalStorageClass,T>` to avoid the collisions

Reply:
Done